client/src/templates/Challenges/components/help-modal.css

```
@@ -13,8 +13,16 @@
   word-spacing: -0.4ch;
 }
 
+.chars-left {
+  padding-top: 10px;
+}
+
 @media screen and (max-width: 767px) {
   .help-modal .btn-lg {
     font-size: 16px;
   }
 }
+
+a {
+  text-align: left;
+}
```

```suggestion
}

.help-modal-body legend {
  color: var(--secondary-color);
}
```

Fix the added text above the checkboxes on the second dialog.